frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/error-components/utils.ts

```diff
@@ -163,3 +167,30 @@ export const computeStorageErrors = (state, wizardReduxID) => {
 
   return { hasAllRequiredFilled, isValid, errors };
 };
+
+const isPermissionError = (title: string, message: string): boolean =>
+  title.includes('roles.rbac.authorization.k8s.io') ||
```

Comment:
this is only a specific error for operating on roles. IMO we should check the http code and mention 403 all errors.

Reply:
agree.